app/migrations/versions/f63f9cf579c1_.py

```diff
@@ -1,16 +1,16 @@
 """empty message
 
-Revision ID: 097d6eedce34
+Revision ID: f63f9cf579c1
 Revises: 
-Create Date: 2021-09-18 07:55:41.307362
+Create Date: 2021-09-20 08:27:20.487921
 
 """
 from alembic import op
 import sqlalchemy as sa
 
 
 # revision identifiers, used by Alembic.
-revision = '097d6eedce34'
+revision = 'f63f9cf579c1'
```

Comment:
Error in migration. Wrong spelling

Reply:
Good catch. Latest migration should have fixed the issue